Makefile

```diff
@@ -10,10 +15,15 @@ fmt:
 lint:
 	cargo clippy -- -D warnings
 
+.PHONY: e2e-tests
+e2e-tests: annotated-policy.wasm
+	bats e2e.bats
```

Comment:
Should not we add a fake test file to avoid errors right after the users create their projects?